repository-data/webfiles/src/main/resources/site/scss/components/_site-header.scss

```diff
@@ -1,46 +1,213 @@
 @import "../environment/tools/mixins/static-button";
+@import "../environment/tools/functions/easing";
 
-/* Site header */
-.site-header {
-    position: relative;
-    border-bottom: 1px solid $col-primary;
-    padding: round($gutter * 1.375) 0;
-    vertical-align: bottom;
+$burger-piece-width: 32px;
+$burger-piece-height: 3px;
+
+@mixin berger-menu(){
```

Comment:
Was this a typo on `burger-menu` or a purposeful spelling?

Reply:
No, just not very good at spotting spelling errors :) thanks for spotting the typo.